core/src/test/java/io/confluent/rest/SaslTest.java

```diff
@@ -163,14 +163,14 @@ public void testNoAuthAttemptOnWs() throws Exception {
   @Test
   public void testBadLoginAttempt() throws Exception {
     try (CloseableHttpResponse response =
-        makeGetRequest("/test", "this shouldnt work")) {
+        makeGetRequest("/test", "dGVzdA==")) {
```

Comment:
what is dGVzdA== ?

Reply:
Its string “test” encoded in base64 format.  In newer version, jetty basic auth validates base63 string format. current string is not a valid base64 string,